core/include/tangram/map.h

```diff
@@ -100,6 +100,18 @@ struct CameraPosition {
     float tilt = 0;
 };
 
+struct EdgePadding {
+    int left = 0;
```

Comment:
Can we have negative padding? Or rather should we allow negative padding?

Reply:
Yeah it's possible. It's not very useful but some application might need it - might as well allow it.